### PR TITLE
Fix error in setup_env.ps1

### PR DESCRIPTION
**Pull request by backwind1233**
as title

### CODE REVIEW

**Comment by backwind1233** [comment]
@chenrujun  @hui1110  please help review this PR

File: servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-topic/terraform/setup_env.ps1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$env:SERVICEBUS_NAMESPACE_CONNECTION_STRING=$(terraform -chdir=terraform output -raw SERVICEBUS_NAMESPACE_CONNECTION_STRING)
+$env:SERVICEBUS_NAMESPACE_CONNECTION_STRING=$(terraform -chdir=terraform output -raw CONNECTION_STRING)
```
Comment (chenrujun) [inline servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-topic/terraform/setup_env.ps1:1]: How about keep same style?

1. https://github.com/Azure-Samples/azure-spring-boot-samples/blob/spring-cloud-azure_4.0.0-beta.4/servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-topic/terraform/outputs.tf#L1
2. https://github.com/Azure-Samples/azure-spring-boot-samples/blob/spring-cloud-azure_4.0.0-beta.4/servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-queue/terraform/outputs.tf#L1

Comment (backwind1233) [reply servicebus/spring-cloud-azure-starter-servicebus-jms/servicebus-jms-topic/terraform/setup_env.ps1]: updated

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**